src/app/api/places/image/route.ts

```diff
@@ -0,0 +1,118 @@
+import axios, { AxiosError } from "axios";
+import { NextRequest } from "next/server";
+import { OpenAI } from "openai";
+import { z } from "zod";
+
+import env from "@/config/env.config";
+import { AgeRangeEnum, GenderEnum, TimeSlots } from "@/enums/app.enum";
+import { apiHandler } from "@/helpers/api-handler";
+import { CategoryEnum } from "@/schemas/activity-categorization-schema";
+import { FreepikRequestConfig, FreepikResponse } from "@/types/freepik.type";
+
+const client = new OpenAI({
+  apiKey: env.openai.apiKey,
+});
+
+const querySchema = z.object({
+  age_range: z.nativeEnum(AgeRangeEnum),
+  gender: z.nativeEnum(GenderEnum),
+  place_category: z
+    .string()
+    .transform((val) => parseInt(val, 10))
+    .pipe(z.nativeEnum(CategoryEnum)),
+  time_slot: z.nativeEnum(TimeSlots),
+});
+
+export const generateBackgroundFeaturesAndCharacterAppearance = async (
```

Comment:
![CleanShot 2024-12-26 at 9  55 35@2x](https://github.com/user-attachments/assets/4334d450-5f3c-48a3-b897-997f5fa86ebe)

The reason for failing is that this file `route.ts` should not export any function or variables that are not allowed than nextjs specification.
ref. https://nextjs.org/docs/app/api-reference/file-conventions/route